apps/docs/content/guides/platform/credits.mdx

```diff
@@ -0,0 +1,60 @@
+---
+id: 'credits'
+title: 'Credits'
+---
+
+## Credit balance
+Each organization has a credit balance. Credits are applied to future invoices to reduce the amount due. As long as the credit balance is greater than $0, credits will be used before charging your payment method on file.
+
+<Image
+  alt="Subscription upgrade modal"
+  src={{
+    light: '/docs/img/guides/platform/credit-balance--light.png',
+    dark: '/docs/img/guides/platform/credit-balance--dark.png',
+  }}
+  zoomable
+/>
+
+You can find the credit balance on the [organization's billing page](https://supabase.com/dashboard/org/_/billing).
+
+### What causes the credit balance to change?
+**Subscription plan downgrades:** Upon subscription downgrade, any prepaid subscription fee will be credited back to your organization for unused time in the billing cycle.\
+As an example, if you start a Pro Plan subscription on January 1 and downgrade to the Free Plan on January 15, your organization will receive approximately $12.50 in credits for the unused time between January 15 and January 31.
+
+**Credit top-ups:** For more information, see [Credit top-ups](/docs/guides/platform/credits#how-to-top-up-your-credits)
+
+## Credit top-ups
+You can top up credits at any time, with a maximum of $999 per top-up. These credits do not expire and are non-refundable.
+
+You may want to consider this option to avoid issues with recurring payments, gain more control over how often your credit card is charged, and potentially make things easier for your accounting department.
+
+<Admonition type="info">
+  If you are interested in larger (> $1,000) credit packages, please [reach out](https://supabase.com/dashboard/support/new?subject=I%20would%20like%20to%20inquire%20about%20larger%20credit%20packages&category=Billing).
+</Admonition>
+
+### How to top up credits
+1. On the [organization's billing page](https://supabase.com/dashboard/org/_/billing), go to section **Credit Balance**
+2. Click **Top Up**
+3. Choose the amount
+4. Choose a payment method or add a new payment method
+5. Click **Top Up**
+
+<Image
+  alt="Subscription upgrade modal"
+  src={{
+    light: '/docs/img/guides/platform/credit-top-up--light.png',
+    dark: '/docs/img/guides/platform/credit-top-up--dark.png',
+  }}
+  zoomable
+  className="max-w-[500px]"
+/>
+
+## Credit FAQ
+### Can I transfer credits to another organization?
+Yes, you can transfer credits to another organization. Please submit a [support ticket](https://supabase.help).
+
+### Can I get a refund of my unused credits?
+No, we do not provide refunds. Please refer to our [Terms of Service](https://supabase.com/terms#:~:text=5.-,Fees%20and%20Taxes.,-1.%20Fees.).
```

Comment:
Would be good to put anchor links in the ToS for use cases like this. Links to highlights only work in Chrome, and only as long as we don't change the target text.

Reply:
Ah, wasn't aware of that. Just copied that over from the FAQs. Changed it to anchors in both places.